### PR TITLE
Added empty state in card to empty state guidelines

### PR DESCRIPTION
**Pull request by mmenestr**
@mcarrano let me know if what I added looks good to you. It's at the very bottom of the guidelines, and I tried to address the feedback we got at the PF share!

### CODE REVIEW

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md [hunk 1]
```diff
@@ -53,27 +53,29 @@ Every data list can be extended with these functionalities:
...
 
 ### Expandable data list
 The expandable list adds an expansion panel to every row to reveal more details about the item.
 
-<img src="./img/expandable-list.png" alt="Expandable data list" /> 
+<img src="./img/expandable-data-list.png" alt="Expandable data list" width="1241"/> 
 
-1. **Expansion caret:** Toggles the expansion open and closed.
-2. **Expansion panel:** A container that is revealed when the expansion is open. It can accept any supported layout to present additional content to the user.
+1. **Expand all:** Expands or collapses every row in the data list at the same time.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/components/data-list/data-list.md:76]: Should this be marked as 'optional'? I don't think you would always require this feature.

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md [hunk 2]
```diff
@@ -53,27 +53,29 @@ Every data list can be extended with these functionalities:
...
+The actionable data list provides checkboxes or radio buttons that enable users to select one or more rows in the list. Users may then act on those selections using options in the [toolbar](/components/toolbar/design-guidelines).
 
-<img src="./img/actionable-list.png" alt="Actionable list" /> 
+<img src="./img/actionable-data-list.png" alt="Actionable list" /> 
 
 1. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information. 
-2. **Checkbox:** Enables a user to select a row.
-3. **Global actions:** Actions that can be applied to all selected items.
+2. **Checkbox:** Enables a user to select a row. Use when multiple rows can be selected at the same time.
+3. **Radio button:** Enables a user to select a row. Use when only one row can be selected at a time.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/components/data-list/data-list.md:62]: Even though this seems perfectly valid, we didn't actually implement this feature for data lists.

File: packages/v4/src/content/design-guidelines/components/empty-state/empty-state.md [hunk 3]
```diff
@@ -189,6 +189,10 @@ In some situations, users may need to add or create something to view associated
 
 <img src="./img/add-or-create.png" alt="Example of add empty state" width="990"/> 
 
+You may also use an extra-small empty state inside of a card to present the user with another meands of adding a new card/item to their card view. If used, we recommend using it **in addition to** a primary button in a toolbar, to ensure that the user is still able to perform the action no matter what page of the card view they are on. We also recommend placing the addition action card as the very first or as the very last card in the card view.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/components/empty-state/empty-state.md:192]: Maybe add that placement should be consistent with where a new card will be added.

File: packages/v4/src/content/design-guidelines/components/table/table.md [hunk 4]
```diff
@@ -47,10 +47,11 @@ Every table can be extended with these functionalities:
 
 ### Expandable table
 
-<img src="./img/expandable-data-table.png"  alt="Expandable table"  width="1186"/> 
+<img src="./img/expandable-table.png"  alt="Expandable table"  width="1186"/> 
 
-1. **Expansion:** Expands the row
-2. **Expansion panel:** Contains details associated with a row. See [expanded panel coloring](#expanded-panel-coloring) for more information.
+1. **Expand all**: Expands every row at the same time.
```
Comment (mcarrano) [inline packages/v4/src/content/design-guidelines/components/table/table.md:52]: Should be optional?

**Review by mcarrano — CHANGES_REQUESTED** [review]
@mmenestr Looks good. See my comments. I'm also wondering if we should add something about using the empty state card in the Card View documentation here: https://patternfly-org-pr-2294-v4.surge.sh/v4/components/card/design-guidelines#card-view-usage That's where I would have expected to look for it.

**Comment by mmenestr** [comment]
@mcarrano please review again. I added the empty state card to the card view guidelines, and removed the radio button mention from the data list! + added "optional" for the expand all!

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md (hunk 2 above)
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/components/data-list/data-list.md]: Removed it from data list then!

**Review by mcarrano — APPROVED** [review]
Looks good @mmenestr . Thanks for making these changes. I will merge.

**Pull request merged**